diff --git a/CadDataTypes/CadDataTypes.csproj b/CadDataTypes/CadDataTypes.csproj
index 90291ed..30833ac 100644
--- a/CadDataTypes/CadDataTypes.csproj
+++ b/CadDataTypes/CadDataTypes.csproj
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
     <Platforms>AnyCPU;x64;x86</Platforms>
   </PropertyGroup>
 
diff --git a/GeoLibWrapper/GeoLibWrapper.vcxproj b/GeoLibWrapper/GeoLibWrapper.vcxproj
index bcefa77..cf15cc8 100644
--- a/GeoLibWrapper/GeoLibWrapper.vcxproj
+++ b/GeoLibWrapper/GeoLibWrapper.vcxproj
@@ -32,7 +32,7 @@
     <ProjectGuid>{9b7a57c2-d91e-4692-a2bc-ed6018e3d2f0}</ProjectGuid>
     <RootNamespace>GeoLibWrapper</RootNamespace>
     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
     <TargetFrameworkVersion>
     </TargetFrameworkVersion>
   </PropertyGroup>
@@ -40,21 +40,21 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
     <CharacterSet>Unicode</CharacterSet>
     <CLRSupport>NetCore</CLRSupport>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
     <CharacterSet>Unicode</CharacterSet>
     <CLRSupport>NetCore</CLRSupport>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <CharacterSet>Unicode</CharacterSet>
     <CLRSupport>NetCore</CLRSupport>
@@ -62,7 +62,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <CharacterSet>Unicode</CharacterSet>
     <CLRSupport>NetCore</CLRSupport>
@@ -70,14 +70,14 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
     <CharacterSet>Unicode</CharacterSet>
     <CLRSupport>NetCore</CLRSupport>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <CharacterSet>Unicode</CharacterSet>
     <CLRSupport>NetCore</CLRSupport>
diff --git a/TCad/TCad.csproj b/TCad/TCad.csproj
index dedddd9..b965103 100644
--- a/TCad/TCad.csproj
+++ b/TCad/TCad.csproj
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net6.0-windows</TargetFramework>
+    <TargetFramework>net5.0-windows</TargetFramework>
     <UseWPF>true</UseWPF>
     <UseWindowsForms>true</UseWindowsForms>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
diff --git a/TestApp/TestApp.csproj b/TestApp/TestApp.csproj
index 4c29d97..df26259 100644
--- a/TestApp/TestApp.csproj
+++ b/TestApp/TestApp.csproj
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0-windows</TargetFramework>
+    <TargetFramework>net5.0-windows</TargetFramework>
     <PlatformTarget>x86</PlatformTarget>
   </PropertyGroup>
 
diff --git a/build_for_net5.0.patch b/build_for_net5.0.patch
index f63c65d..add04fa 100644
--- a/build_for_net5.0.patch
+++ b/build_for_net5.0.patch
@@ -1,75 +0,0 @@
-diff --git a/CadDataTypes/CadDataTypes.csproj b/CadDataTypes/CadDataTypes.csproj
-index e5f0942..a6cc7f8 100644
---- a/CadDataTypes/CadDataTypes.csproj
-+++ b/CadDataTypes/CadDataTypes.csproj
-@@ -1,7 +1,7 @@
- ﻿<Project Sdk="Microsoft.NET.Sdk">
- 
-   <PropertyGroup>
--    <TargetFramework>net6.0</TargetFramework>
-+    <TargetFramework>net5.0</TargetFramework>
-     <Platforms>AnyCPU;x64;x86</Platforms>
-   </PropertyGroup>
- 
-diff --git a/GeoLibWrapper/GeoLibWrapper.vcxproj b/GeoLibWrapper/GeoLibWrapper.vcxproj
-index bcefa77..75be6c7 100644
---- a/GeoLibWrapper/GeoLibWrapper.vcxproj
-+++ b/GeoLibWrapper/GeoLibWrapper.vcxproj
-@@ -32,7 +32,7 @@
-     <ProjectGuid>{9b7a57c2-d91e-4692-a2bc-ed6018e3d2f0}</ProjectGuid>
-     <RootNamespace>GeoLibWrapper</RootNamespace>
-     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
--    <TargetFramework>net6.0</TargetFramework>
-+    <TargetFramework>net5.0</TargetFramework>
-     <TargetFrameworkVersion>
-     </TargetFrameworkVersion>
-   </PropertyGroup>
-diff --git a/GeoLibWrapper/cad_data_types.h b/GeoLibWrapper/cad_data_types.h
-index f128cd5..2d77336 100644
---- a/GeoLibWrapper/cad_data_types.h
-+++ b/GeoLibWrapper/cad_data_types.h
-@@ -1,14 +1,14 @@
- #pragma once
- #ifdef WIN32
- 	#ifdef _DEBUG
--		#using "..\\CadDataTypes\\bin\\x86\\Debug\\net6.0\\CadDataTypes.dll"
-+		#using "..\\CadDataTypes\\bin\\x86\\Debug\\net5.0\\CadDataTypes.dll"
- 	#else
--		#using "..\\CadDataTypes\\bin\\x86\\Release\\net6.0\\CadDataTypes.dll"
-+		#using "..\\CadDataTypes\\bin\\x86\\Release\\net5.0\\CadDataTypes.dll"
- 	#endif
- #else
- 	#ifdef _DEBUG
--		#using "..\\CadDataTypes\\bin\\x64\\Debug\\net6.0\\CadDataTypes.dll"
-+		#using "..\\CadDataTypes\\bin\\x64\\Debug\\net5.0\\CadDataTypes.dll"
- 	#else
--		#using "..\\CadDataTypes\\bin\\x64\\Release\\net6.0\\CadDataTypes.dll"
-+		#using "..\\CadDataTypes\\bin\\x64\\Release\\net5.0\\CadDataTypes.dll"
- 	#endif
- #endif
-diff --git a/TCad/TCad.csproj b/TCad/TCad.csproj
-index 9c83639..46bee37 100644
---- a/TCad/TCad.csproj
-+++ b/TCad/TCad.csproj
-@@ -2,7 +2,7 @@
- 
-   <PropertyGroup>
-     <OutputType>WinExe</OutputType>
--    <TargetFramework>net6.0-windows</TargetFramework>
-+    <TargetFramework>net5.0-windows</TargetFramework>
-     <UseWPF>true</UseWPF>
-     <UseWindowsForms>true</UseWindowsForms>
-     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-diff --git a/TestApp/TestApp.csproj b/TestApp/TestApp.csproj
-index 4c29d97..df26259 100644
---- a/TestApp/TestApp.csproj
-+++ b/TestApp/TestApp.csproj
-@@ -2,7 +2,7 @@
- 
-   <PropertyGroup>
-     <OutputType>Exe</OutputType>
--    <TargetFramework>net6.0-windows</TargetFramework>
-+    <TargetFramework>net5.0-windows</TargetFramework>
-     <PlatformTarget>x86</PlatformTarget>
-   </PropertyGroup>
- 
